core/src/main/java/jenkins/model/Jenkins.java

```diff
@@ -342,7 +342,7 @@
     /**
      * The Jenkins instance startup type i.e. NEW, UPGRADE etc
      */
-    private String installStateName;
+    private transient String installStateName;
```

Comment:
I'm not sure this is a good idea. The way this field worked (or didn't) caused me to abandon work in this area recently.

Wouldn't be surprised if this also causes regressions involving Jenkins being restarted while the setup wizard is active. If we're taking a closer look at this change, I recommend this be checked as well.

Reply:
That case was handled